app/views/jobs/show.html.erb

```diff
@@ -1,13 +1,26 @@
-Title: <%= @job.title %>
+<h4>Title:</h4> <%= @job.title %>
 <br>
 
-Description: <%= @job.description %>
+<h4>Description:</h4> <%= @job.description %>
 <br>
 
-Level of Interest: <%= @job.level_of_interest %>
+<h4>Level of Interest:</h4> <%= @job.level_of_interest %>
 <br>
 
-City: <%= @job.city %>
+<h4>City:</h4> <%= @job.city %>
 <br>
 
-Company: <%= link_to @job.company.name, company_path(@job.company) %>
+<h4>Company:</h4> <%= link_to @job.company.name, company_path(@job.company) %><br>
+
+<%= render partial: 'jobs/comment', collection: @job.comments.order_comments %>
+
+<h4> Add a Comment </h4>
+<%= form_for @comment do |f| %>
+              <%= f.label :comment %>
+              <%= f.text_field :content %>
+              <%= hidden_field(:job, :id) %>
```

Comment:
whenever you find yourself passing data via a hidden field in a form, there's a better way of doing things. in this case, i'd suggest nesting your routes